package.json

```
@@ -30,12 +30,13 @@
   },
   "homepage": "https://github.com/transitmatters/new-train-tracker#readme",
   "dependencies": {
-    "@types/react": "^18.0.1",
+    "@types/react": "^17.0.0",
     "@types/react-dom": "^18.0.0",
```

React and React-DOM should usually be on the same version

```suggestion
    "@types/react": "^17.0.0",
    "@types/react-dom": "^17.0.0",
```